src/vaadin-radio-group.html

```diff
@@ -286,6 +288,14 @@
           });
         }
 
+        _selectIncButton(next, checkedRadioButton) {
+          if (!next) {
```

Comment:
nit: maybe it's better to revert the condition so it would be `if (next)` as that's the default case?

Reply:
Updated.